rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+
+    person.add_rental(self)
```

Comment:
- [ ] Please ensure that the rental is adding itself to the @book achieving the 'belongs-to' association. :pray:
  ```ruby
      book.rentals << self
  ```